src/plots/barPlot.ts

```diff
@@ -113,6 +120,56 @@ export class Bar<X, Y> extends XYPlot<X, Y> {
     return this;
   }
 
+  public barEnd(): Plots.TransformableAccessorScaleBinding<X, number>;
```

Comment:
Is this different enough from setting an `x2`/`y2` on a Rectangle Plot to warrant a new endpoint for bar plots? Feels like the API has a lot of overlap